doc/manual/src/json/experimental-notice.md

```diff
@@ -0,0 +1,5 @@
+> **Warning**
+>
+> This format is currently
+> [**experimental**](@docroot@/contributing/experimental-features.md#xp-feature-nix-command)
```

Comment:
I think this can be put inline, because then we can say which feature exactly. This include seems like more trouble than it's worth.